console-subscriber/Cargo.toml

```diff
@@ -28,6 +28,7 @@ keywords = [
 default = ["env-filter"]
 parking_lot = ["parking_lot_crate", "tracing-subscriber/parking_lot"]
 env-filter = ["tracing-subscriber/env-filter"]
+grpc-web = ["tonic-web"]
```

Comment:
Do you have any opinions about the feature name?

Reply:
I think that `grpc-web` sounds fine.